sdk/cosmos/azure-spring-data-cosmos/pom.xml

```diff
@@ -172,15 +172,6 @@
   </dependencies>
 
   <build>
-    <resources>
-      <resource>
-        <directory>src/main/resources</directory>
-        <filtering>true</filtering>
-        <includes>
-          <include>META-INF/project.properties</include>
```

Comment:
If the META-INF/project.properties was the only file that was being filtered out, then we should still have the `spring.factories` file shipped in the jar. So I am not sure if this would fix the issue or not. 
Did we verify this locally, by creating a jar and making sure these files are now present in the jar ?

Reply:
Yes, tested.

Reply:
@kushagraThapar are we okay to merge this PR now?

Reply:
@saragluna - yes this is good to go in now. Thanks for the reminder.